api/mesh/options.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package kuma.mesh;
+
+option go_package = "github.com/kumahq/kuma/api/mesh";
+
+import "google/protobuf/descriptor.proto";
+
+message KumaResource {
+  // Name of the Kuma resource struct.
+  string name = 1;
+
+  // Name and value of the modelResourceType constant.
+  string type = 2;
+
+  // True if this resoure has global scope. Otherwise it will be mesh scope.
+  bool global = 3;
```

Comment:
can we change this to `scope` which takes values of `mesh` and `global`? This would be more inline with what we have right now in the code.

Reply:
Yeh, the reason I went with this is that I didn't want to add it to every proto, and I didn't want to bind the type names as strings. Is there any reasonable likelihood of creating other kinds of scopes? Maybe we could just change it later if we need to?

Reply:
> Is there any reasonable likelihood of creating other kinds of scopes?

Not that I see at the moment, but I just want to leave room for it.

> I didn't want to add it to every proto

we could also assume that empty scope = mesh scope.

Anyways, I don't require to do this change from my side so it's up to you :)

Reply:
OK, let's leave it as is for now. Happy to revisit in the future.